src/components/Coverage.tsx

```diff
@@ -56,14 +53,13 @@ const Coverage = ({
   selectedOutlineColor = 'white',
   fillOpacity = 0.5,
   onHexSelected,
-  visible = true,
   selectedHexId,
   outlineWidth = 2,
   selectedOutlineWidth = 5,
-  outline = true,
-  showCount = false,
   witnesses,
 }: Props) => {
+  useEffect(() => {}, [witnesses])
+
   const boundingBox = useMemo(() => {
     return boundsToFeature(bounds)
```

Comment:
This is where I added a check for `if (zoomLevel < 11) return { type: 'Feature' } as Feature` which helps with performance but then you have to set up an effect to keep track of the zoomLevel

Reply:
I also added `    if (!bounds || !visible || zoomLevel < 11)
      return { type: 'FeatureCollection', features: [] } as FeatureCollection` in the source set function